### PR TITLE
[fixes/bv-106] barchart data

### PR DESCRIPTION
**Pull request by dariuszsobkowicz**
- fix fetching barchart data
- fix tabs typography and table blockchain row name
- fix duplicating http requests
- fix routing

### CODE REVIEW

File: src/components/DashboardTable/DashboardTable-container.tsx [hunk 1]
```diff
@@ -1,17 +1,23 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+
 import DashboardTableView from './DashboardTable-view';
 
-const createData = (name: string, icon: string ) => ({ name, icon })
+import { Table } from './types';
+
+const DashboardTable = (props: any) => {
```
Comment (agatascalac) [inline src/components/DashboardTable/DashboardTable-container.tsx]: A co to? Any 😃

File: src/components/DashboardTable/DashboardTable-container.tsx [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
+  const { match: { params: { blockchains } } } = props;
+
+  const blockchainNamesMap: any = {
```
Comment (agatascalac) [inline src/components/DashboardTable/DashboardTable-container.tsx:10]: j/w

**Review by agatascalac — APPROVED** [review]

File: src/components/DashboardTable/DashboardTable-container.tsx (hunk 1 above)
Comment (dariuszsobkowicz) [reply src/components/DashboardTable/DashboardTable-container.tsx]: 🤷‍♂️

File: src/components/DashboardTable/DashboardTable-container.tsx (hunk 2 above)
Comment (dariuszsobkowicz) [reply src/components/DashboardTable/DashboardTable-container.tsx:10]: 🤷‍♂️

Comment (Milena-Wasiak) [reply src/components/DashboardTable/DashboardTable-container.tsx:10]: interfaceDareczka {
name:string,
icon: string
}

albo iconę nazwij `ethereum`, wywal element icon. i otypuj Record<string, string> 

:)

**Review by Milena-Wasiak — APPROVED** [review]

**Pull request merged**